src/set_theory/surreal/infinitesimal.lean

```diff
@@ -0,0 +1,153 @@
+/-
+Copyright (c) 2022 Violeta Hernández Palacios. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Violeta Hernández Palacios
+-/
+
+import set_theory.surreal.dyadic
+
+/-!
+# Infinitesimal pre-games
+
+We define the notions of an infinitesimal and a strongly infinitesimal pre-game, and prove the basic
+results.
+
+## Main definitions
+
+- `pgame.infinitesimal`: a pre-game between `-2 ^ (-n)` and `2 ^ (-n)` for any `n : ℕ`.
+- `pgame.strong_infinitesimal`: a pre-game between `-x` and `x` for any positive numeric `x`.
```

Comment:
Which one of the `infinitesimal` definitions is `small`? And what would the name be for the other?

Reply:
`strong_infinitesimal` is `small`, and `infinitesimal` is just `infinitesimal`, in ONAG and CGT. (ONAG actually uses real numbers instead of dyadics to define `infinitesimal`, but that's equivalent, and we don't have the embedding from reals to surreals/games yet.)

Reply:
I've applied the rename.